src/consts/Stages.ts

```diff
@@ -14,8 +14,8 @@ interface STARTSTAGE {
 interface MIDSTAGE {
   key: string;
   text: string;
-  next: string;
-  nextEventTitle: string;
+  next: (request: RPARequest | undefined) => string | undefined;
+  nextEventTitle: (request: RPARequest | undefined) => string | undefined;
```

Comment:
Just make it an optional parameter?
```suggestion
  next: (request?: RPARequest) => string | undefined;
  nextEventTitle: (request?: RPARequest) => string | undefined;
```